spring-integration-http/src/main/java/org/springframework/integration/http/outbound/HttpRequestExecutingMessageHandler.java

```diff
@@ -79,40 +52,12 @@
  * @author Gunnar Hillert
  * @author Artem Bilan
  * @author Wallace Wadge
+ * @author Shiliang Li
  * @since 2.0
  */
-public class HttpRequestExecutingMessageHandler extends AbstractReplyProducingMessageHandler {
-
-	private final Map<String, Expression> uriVariableExpressions = new HashMap<>();
-
+public class HttpRequestExecutingMessageHandler extends AbstractHttpRequestExecutingMessageHandler {
```

Comment:
Well, from big height it looks like we are ready to merge.
But it seems for me that you don't use Git's `rebase` functionality.

This PR can't be merged properly to the `master` because a lot of conflicts.
And this class is the worst.

Please, make a `git rebase master`, resolve conflicts and just push it to the `origin`.

Thank you for understanding.